torch/onnx/utils.py

```diff
@@ -1314,6 +1314,70 @@ def _validate_dynamic_axes(dynamic_axes, model, input_names, output_names):
             dynamic_axes[key] = value_dict
 
 
+# This function export torch::jit::Graph object
```

Comment:
please change from a comment above the function to a doc string inside the function